src/shogun/base/SGObject.h

```diff
@@ -59,50 +60,18 @@ template <class T> class SGStringList;
  * Macros for registering parameters/model selection parameters
  ******************************************************************************/
 
-#ifdef _MSC_VER
-
-#define VA_NARGS(...)  INTERNAL_EXPAND_ARGS_PRIVATE(INTERNAL_ARGS_AUGMENTER(__VA_ARGS__))
```

Comment:
While it would be good to get rid of this thing, I think we still need to keep it (see below on default properties)

Reply:
OK, I will bring this back and add a SG_ADD3!